WASM/basic_template.html

```diff
@@ -1,77 +1,114 @@
-
 <!doctype html>
 <html lang="en-us">
-    <head>
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <meta charset="utf-8">
-        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-        <title>Emscripten-Generated Code</title>
-        <style>
-html,body {	width: 100%; height: 100%; }
-body { font-family: arial; margin: 0; padding: 0; background: #000; }
-
-.emscripten { padding-right: 0; margin-left: auto; margin-right: auto; display: block; }
-div.emscripten_border { border: none; }
 
-/* the canvas *must not* have any border or padding, or mouse coords will be wrong */
-canvas.emscripten { border: 0px none; background-color: black; }
-        </style>
-    </head>
-    <body>
-        <canvas class="emscripten" id="canvas" oncontextmenu="event.preventDefault()" tabindex=-1></canvas>
-        <script type='text/javascript'>
-var Module = {
-    preRun: [],
-    postRun: [],
-    canvas: (function() {
-        var canvas = document.getElementById('canvas');
+<head>

```

Comment:
Can you fix the indentation here, please, so it's easier to see your actual change?

Reply:
Yeah I will try to